api/src/bigquerytest/java/org/pmiops/workbench/api/CohortBuilderControllerBQTest.java

```diff
@@ -199,6 +209,20 @@ public void setUp() {
 
     cdrVersion = cdrVersionDao.save(cdrVersion);
 
+    DbWorkspace dbWorkspace = new DbWorkspace();
+    dbWorkspace.setWorkspaceNamespace(WORKSPACE_NAMESPACE);
+    dbWorkspace.setName("Saved workspace");
+    dbWorkspace.setFirecloudName(WORKSPACE_ID);
+    dbWorkspace.setCdrVersion(cdrVersion);
+
+    workspaceDao.save(dbWorkspace);
+    when(workspaceAuthService.enforceWorkspaceAccessLevel(
+            WORKSPACE_NAMESPACE, WORKSPACE_ID, WorkspaceAccessLevel.READER))
+        .thenReturn(WorkspaceAccessLevel.READER);
```

Comment:
We shouldn't need to mock this, since it isn't called in the CohortBuilderController